sql/src/test/java/org/apache/druid/sql/avatica/DruidAvaticaHandlerTest.java

```diff
@@ -1097,9 +1101,24 @@ public void testParameterBinding() throws Exception
   }
 
   @Test
-  public void testSysTableParameterBinding() throws Exception
+  public void testSysTableParameterBindingRegularUser() throws Exception
   {
-    PreparedStatement statement = client.prepareStatement("SELECT COUNT(*) AS cnt FROM sys.servers WHERE servers.host = ?");
+    PreparedStatement statement =
+        client.prepareStatement("SELECT COUNT(*) AS cnt FROM sys.servers WHERE servers.host = ?");
```

Comment:
is it a bug that this didn't explode before or is it an artifact of changes elsewhere or just a flaw in the original test?

Reply:
ah nevermind, found the change in `CalciteTests` that limits to datasource and view permissions

Reply:
Ah, yeah, this is happening because I made the test authorizer more strict for the "regular user". The regular user now only has permissions to datasources and views, not state or external data.